sky/packages/sky/lib/README.md

```diff
@@ -1,71 +0,0 @@
-SKY SDK
```

Comment:
I have it locally, but I haven't pushed it yet.  I was going to replace the ascii art with a link to the flutter architecture slides before pushing it.